packages/indexer-native/native/Cargo.toml

```diff
@@ -6,9 +6,10 @@ authors = [
   "Theo Butler <theodusbutler@gmail.com>",
 ]
 license = "MIT"
-edition = "2018"
+edition = "2021"
```

Comment:
What's the importance of 2021 here?

Reply:
It's not important if we're retiring this... but keeping this dialect up to date is nice if we weren't.